source/getting_started/concepts.rst

```diff
@@ -1,120 +1,42 @@
-The actual information, though, is the true information
-about whether the patient has heart disease in real life. If the patient does, in fact, have heart disease, then the conviction value of the positive prediction would be around one,
-as the patient's actual health information and diagnosis status are similar to what is expected.
-However, if the patient actually does not have heart disease, the actual heart disease diagnosis prediction is not what was expected, so the conviction would be low, indicating a surprising outcome.
-While this is an example, the concept of surprisal,
-quantified by conviction, is prevalent throughout many of Howso's analysis capabilities and can be adapted for a variety of tools and use cases.
-
-Basic Workflows
-^^^^^^^^^^^^^^^
-
-There are three main components of a Howso Engine workflow: building a `Trainee`, `analyzing` the data, and `reacting` to new data.
+    Unlike other AI tools, Howso calculates how much influence each original data point has on a prediction. This influence is related to the probability that the data point is representative of
+    the prediction. Influence weights are computed across all features and then aggregated per data point; the most influential data have the highest total influence weights. Being able to see the
+     influence of each data point
+    on a decision provides direct evidence for why a decision was made. 
 
-Trainee
--------
+- Outperform other commonly used feature importance metrics, including SHAP. ​
 
-A `Trainee` is a collection of data, metadata, and hyperparameters upon which the IBL algorithm acts. This is analogous to a model in traditional machine learning settings, but is
-different in that it is the *actual data*, stored in memory, and not an abstract representation of the data.
+    Howso quantifies individual feature contributions to a prediction, i.e., how much an individual feature impacts a prediction. The concept of feature contribution is similar to the data science concept of "feature importance". However,
+    Howso is robust against several common challenges (correlated features, redundant features, difference in scale between features, and multiple distinguishing features) faced by other feature importance tools, 
+    including the SHAP metric, which often lead to misleading results.
 
-Analyze
--------
+- Know Engine's confidence in each decision.​
 
-Once a `Trainee` is built, i.e., the data is stored, it is `analyzed` to understand the relationships between the individual data points. Here, the internal parameters describing these
-relationships are tuned to improve performance and the accuracy of predictions and metrics.
+    Other AI hallucinates, predicting false information with high confidence, because it can't accurately calculate uncertainties. Howso understands what it should know about the uncertainty of the data, and uses
```

Comment:
"Other AI hallucinates" comes off very aggressive to me, and maybe a bit too general. I suggest something like Deep Learning models, Deep Neural Networks, etc rather than "Other AI"

Reply:
Perhaps something like:

"Deep learning methods are often found to confidently predict and present patently false information as though it were true -- informally known as 'hallucinations' -- because these methods cannot accurately calculate uncertainty. In contrast, Howso accurately measures the confidence level of any predictions made and provides these metrics transparently."

Reply:
Good suggestions, thank you!